src/sentry/templates/sentry/bases/react_pipeline.html

```diff
@@ -0,0 +1,40 @@
+{% extends "sentry/layout.html" %}
+{% load sentry_api %}
+{% load sentry_assets %}
+{% block body %}
+<div id="blk_router">
+  <div class="loading triangle">
+    <div class="loading-mask"></div>
+
+    <div class="loading-indicator">
+      <!-- If you update this also its sibling in loadingIndicator.tsx and app/index.html -->
+      <svg width="90" height="90" viewBox="0 0 90 90">
+        <defs>
+          <path
+            id="loading-logo"
+            fill="none"
+            stroke-width="5"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+            d="M8.07 50.083c-1.744 2.885-3.192 5.403-4.344 7.556-1.729 3.228.522 6.041 3.335 6.041h18.86c.056-10.925-6.902-18.96-12.867-22.226 1.9-3.173 5.12-8.986 6.313-10.984 12.02 6.966 19.14 19.803 19.14 33.21 4.01.027 8.422 0 12.638 0 .01-16.823-8.633-34.304-25.512-44.004 4.289-7.375 7.02-12.354 8.532-14.733 1.512-2.378 5.214-2.395 6.656 0 1.443 2.395 28.98 50.16 30.451 52.696 1.471 2.536.034 6.041-3.751 6.041h-6.789"
+          />
+        </defs>
+        <use x="5" y="5" xlink:href="#loading-logo" class="logo-spin-1" />
+        <use x="5" y="5" xlink:href="#loading-logo" class="logo-spin-0" />
+      </svg>
+    </div>
+
+    <div class="loading-message">
+      <p>Please wait while we load an obnoxious amount of JavaScript.</p>
+      <p><small>You may need to disable adblocking extensions to load Sentry.</small></p>
+    </div>
+  </div>
+</div>
+<script>
+  $(function() {
+    if (typeof RenderPipelineView === 'function') {
+        RenderPipelineView("{{ pipelineName }}", {{ props | safe }});
+    };
+  });
+</script>
+{% endblock %}
```

Comment:
So I think you could make a base template for this and extend from it, and have like a `block body_app_boot`